app/views/home/download.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1 class="heading-large">Preview and download your code</h1>
+<p>Your picker and dataset is available as two files:</p>
+<ul class="list">
+  <li>picker.html</li>
+  <li>data.json</li>
+</ul>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <details class="details--with-border">
```

Comment:
The design is more than just a `<hr>` i.e. more spacing so styling is the best option. IMO